lib/core/capabilities/ManualControlCapability.js

```diff
@@ -25,10 +25,9 @@ class ManualControlCapability extends Capability {
     /**
      * @abstract
      * @param {string} action
-     * @param {number} [duration] - duration in milliseconds
      * @returns {Promise<void>}
      */
-    async manualControl(action, duration) {
+    async manualControl(action) {
```

Comment:
> Fix a bunch of issues with base manual control base capability (including removing duration parameter since one would actually need to stop it manually to perform it)

Please explain further

Reply:
I had added the parameter because I inspected python-miio, I saw it supports the duration parameter so I assumed it's an actual feature.

It turns out that this is not the case, python-miio actually sleeps that amount of time and then sends a stop command.
Implementing the same in valetudo doesn't seem ideal.